src/helpers/buildWalletSections.tsx

```diff
@@ -242,9 +242,8 @@ const withBriefBalanceSavingsSection = (
   if (network !== Network.mainnet) {
     return [];
   }
-  // TODO: make this logic conditional
-  return [];
-  if (isLoadingAssets) return [];
+
+  if (isLoadingAssets || totalUnderlyingNativeValue === '0') return [];
```

Comment:
since `totalUnderlyingNativeValue` is a string is it possible that a zero value could be `'0.0'`? maybe better to convert to number and then check if zero

Reply:
good point, we're checking based on the default value set earlier in the file which is `0` so should suffice